pkg/templates/templating.go

```diff
@@ -59,7 +59,7 @@ func ReplacePlaceHoldersVariables(template string, dico map[string]string, rawEv
 				for k, v := range headers {
 					headerMap[k] = v[0]
 				}
-				val, err := customparams.CelValue(key, rawEvent, headerMap, map[string]string{}, changedFiles)
+				val, err := customparams.Value(key, rawEvent, headerMap, map[string]string{}, changedFiles)
```

Comment:
here still https://github.com/openshift-pipelines/pipelines-as-code/pull/1705/files?show-viewed-files=true&file-filters%5B%5D=#diff-416b5f5ac3114bc58b0bcc0b4ec86159ab3032ff2cd90ecc64f537c2a77c70deR32  old Function name is used in the comments

Reply:
I don't see it with this link